### ISSUE TITLE
underground barrier=lift_gate

### ISSUE THREAD

**Issue by maro-21** [gravitystorm/openstreetmap-carto#4098]
An [underground lift gate](https://www.openstreetmap.org/node/2395969584) is rendered although it's on layer=-1 and is a part of an [underground road](https://www.openstreetmap.org/way/231190126) which is not rendered.

**Comment by jeisenbe** [comment]
The tag `layer=-1` does not mean that a feature is underground, it only means that it is below the general level of roads in the area. It might be in an open cutting, for example. (In this case you might want to consider adding `location=underground` to make it clear.)

We do not currently consider the `location=` tag when deciding whether to render point barriers like gates. It is helpful to know if a highway is blocked by a gate, even if it is in a tunnel. 

Normally the highway=service tunnel should be shown in this location. It is not shown because it is underneath a highway=pedestrian area, so it is blocked from rendering by this area

Screenshot of area mentioned:
<img width="402" alt="gate-service-tunnel-under-pedestrian-area" src="https://user-images.githubusercontent.com/42757252/77874199-37c48980-7287-11ea-8474-74db24778fdd.png">

This is a known issue, #529 - but it is hard to solve without causing other problems.

**Comment by jeisenbe** [comment]
I'm closing this issue because it appears to be a duplicate of #529 - we actually do want to render tunnels over highway=pedestrian areas, and we should continue to render lift gates in tunnels or at underground parking garage entrances.

**Comment by maro-21** [comment]
Will the lift gate be no longer visible on the map if add tunnel= to it?

**Comment by jeisenbe** [comment]
That does not affect the rendering.
